components/automate-ui/src/app/ui.component.ts

```diff
@@ -54,6 +55,10 @@ export class UIComponent implements OnInit {
     });
   }
 
+  ngAfterViewChecked() {
+    this.cdRef.detectChanges();
```

Comment:
Force change detection run after view check to check if the model has changed.

Reply:
Clever. Seems analogous to doing a `setTimeout` of zero to give things a chance to settle in other scenarios.